mne/io/nihon/nihon.py

```diff
@@ -165,6 +165,11 @@ def _read_21e_file(fname):
                 f"Could not decode 21E file as one of {_encodings}; "
                 f"Default channel names are chosen."
             )
+    else:
+        warn(
+            "Could not find the 21E file containing channel definitions; "
```

Comment:
At least this to make clear it's an extension
```suggestion
            "Could not find the .21E file containing channel definitions; "
```
or could do
```
f"... {fname.with_suffix('.21E')} ..."
```
or similar